### ISSUE TITLE
Picovoice Issue: 

### ISSUE THREAD

**Issue by kcoderhtml** [Picovoice/picovoice#244]
Make sure you have read the documentation, and have put forth a reasonable effort to find an existing answer.

### Expected behavior
Picovoice to continue running

### Actual behavior
Picovoice crashed with error message.
```
Traceback (most recent call last):
  File "/home/kierank/.local/lib/python3.9/site-packages/picovoice/picovoice.py", line 249, in process
    self._inference_callback(inference)
  File "/home/kierank/Documents/Projects/python/porqupine/python-t2s.py", line 115, in _inference_callback
    actionhandeler(inference.intent, value)
UnboundLocalError: local variable 'value' referenced before assignment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kierank/Documents/Projects/python/porqupine/python-t2s.py", line 240, in <module>
    main()
  File "/home/kierank/Documents/Projects/python/porqupine/python-t2s.py", line 224, in main
    PicovoiceDemo(
  File "/home/kierank/Documents/Projects/python/porqupine/python-t2s.py", line 141, in run
    self._picovoice.process(pcm)
  File "/home/kierank/.local/lib/python3.9/site-packages/picovoice/picovoice.py", line 250, in process
    except pvporcupine.RhinoError as e:
AttributeError: module 'pvporcupine' has no attribute 'RhinoError'
```

### Steps to reproduce the behavior
Run python demo for a while then try to talk to it.

**Comment by kenarsa** [comment]
`AttributeError: module 'pvporcupine' has no attribute 'RhinoError'` is a bug and it should be fixed now with [this commit](24ee934a0597c7b20043881bcd96832d2cbce9e8)

Now the reason you get the exception is unclear please update and run again and report back so I can help.

**Comment by kcoderhtml** [comment]
Thank you!
